### DISCUSSION TITLE
How can I disable/remove the shadow of hover text

### DISCUSSION THREAD

**Discussion by wangxiaochuTHU** [emilk/egui#4083] [Q&A] [answered]
I used `on_hover_text_at_pointer` for showing some hovering text, but it looks that a shadow is also shown around the hovering text. I've tried to explicit set the shadow to `Shadow::NONE`, but that didn't help. So how can I disable the shadow? Thank you.

![image](https://github.com/emilk/egui/assets/86735730/44d85139-5c0a-4955-aa6d-3bac631b6aa8)

```rust
let frame : Frame = Frame::none().shadow(Shadow::NONE);
egui::CentralPanel::default().frame(frame)
        .show(ctx, |ui| {
        /* painting several widgets here*/
        })
        .response
        .on_hover_text_at_pointer( RichText::new("32872").background_color(Color32::TRANSPARENT) )

```

**Comment by YgorSouza** [comment, marked as answer]
The hover text shadow is controlled by [Visuals::popup_shadow](https://docs.rs/egui/0.26.2/egui/style/struct.Visuals.html#structfield.popup_shadow). It also controls the shadow on context menus and combo box drawers. But the [window shadow](https://docs.rs/egui/0.26.2/egui/style/struct.Visuals.html#structfield.window_shadow) is a separate field.

```rs
ctx.style_mut(|style| style.visuals.popup_shadow = Shadow::NONE);
```
